src/util.js

```diff
@@ -45,6 +59,24 @@ export default class Utils {
         return new Int64(numberValue).toBuffer();
     }
 
+
+    /**
+     * @param {string} ip - a string representation of an ip address.
+     * @return {number} - a 32-bit number where each byte represents an
+     * octect of an ip address.
+     **/
+    static ipToInt(ip: string): number {
+        let ipl = 0;
+        let parts = ip.split('.');
+        for (let i = 0; i < parts.length; i++) {
+            ipl <<= 8;
+            ipl += parseInt(parts[i], 10);
+        }
+        // TODO(oibe) prove that this will never be grater than 2^31 because
```

Comment:
169.x.x.x will be greater that 2^31

you just need to take a complement in this case, 2^32 - ipl